### ISSUE TITLE
Imbalanced

### ISSUE THREAD

**Issue by llVIU** [zifnab06/factorio_mods#8]
Hello, I used this armor several times on several maps and the main issues are:
-overpowered because the space is too large
-overpowered because the cost is too low
-cost makes no sense I get the 200 blue card and 200 lightweight material, that makes sense, it's common requirement for personal equipment and high tier equipment... but a few train wagons, lots of electric furnaces and tanks? Irrelevant, makes no sense. 
-research is quite high and expensive, that's the only thing that balances this armor a bit. The first armor requires metal plates, second requires steel, third requires red cards and blue pots, fourth requires blue cards and electric motors, 5th requires yellow potions (mk2 armor), with each armor the requirements are a bit higher in technology and resources, mk4 armor requires lots of white potions which makes sense but the cost doesn't.

But once I get the armor, It's so much cheaper to just spam cheap low tier equipment and put lots of it inside. 50 basic batteries, 100 mk1 roboports, 5 exo skeletons, 30 nuclear reactors from some other mod (cheap 3x3 reactors requiring uranium), fusion reactors are expensive but due to the vast amount of basic batteries and reactors, I can get around this easily. Heck, even 1-3 fusion reactors is enough to power 50 basic batteries. Now put 10 basic shields and 10 personal lasers and you got yourself absolutely no use for artillery, bazooka, tanks, airplanes, because you got infinite ammo, infinite damage, infinite health/shield from spamming low tier equipment. It mainly comes down to the fact that the armor has too much space. There's no reason for me to bother trying to get personal roboport 2 or battery mk2 or even battery mk3 from some other mod.

So tl;dr make the space smaller, remove train wagon/tank/elec furn requirements and put something like rocket parts made in a rocket silo. We both agree this is a step higher, technology wise. and we make them, even though they can't be handleded in vanilla. Or maybe satellites. Or mk3 modules.
